src/com/duckduckgo/mobile/android/adapters/RecyclerMainFeedAdapter.java

```diff
@@ -103,9 +120,7 @@ public HeaderViewHolder(View itemView) {
             instructionbutton = (Button) itemView.findViewById(R.id.instruction_button);
             dismissImageButton = (ImageButton) itemView.findViewById(R.id.dismiss_image_button);
             viewPager = (ViewPager) itemView.findViewById(R.id.view_pager);
-            adapter = new BannerOnboardingAdapter(fragmentManager);
-            viewPager.setAdapter(adapter);
-            viewPager.setPageTransformer(false, new OnboardingTransformer());
+            viewPager.setAdapter(new BannerOnboardingAdapter());
             viewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
```

Comment:
I haven't dug too deeply, I just noticed the warnings in AS, it could be that I'm on a more recent AS version. I like your idea of upping the api and getting the latest support libs and I'm happy for that to happen in a separate PR.